packages/terra-signature/src/Signature.module.scss

```diff
@@ -8,3 +8,4 @@
     width: 100%;
   }
 }
+/* stylelint-enable */
```

Comment:
We don't need to disable `property-no-unknown`, let's remove these stylelint overrides